src/Components/SettingsModal.jsx

```diff
@@ -328,6 +328,35 @@ export const SettingsModal = () => {
                       </ul>
                     </Typography.Paragraph>
                   </Panel> */}
+                  <Panel header={"Version 1.3.1"} key={"1.3.1"}>
```

Comment:
This is the "currently released version".
```suggestion
                  <Panel header={"Version 2.3.1"} key={"2.3.1"}>
```

Reply:
Ah i was reverting a change to a comment and messed it up, should be fixed now